src/nvim/main.c

```diff
@@ -587,7 +588,7 @@ int main(int argc, char **argv)
   set_vim_var_nr(VV_VIM_DID_ENTER, 1L);
   apply_autocmds(EVENT_VIMENTER, NULL, NULL, false, curbuf);
   TIME_MSG("VimEnter autocommands");
-  if (use_remote_ui) {
+  if (use_remote_ui && !listen_and_embed) {
```

Comment:
This is probably still incorrect, but I will fix it in another PR.

Reply:
ok, thanks!

Reply:
This seems to be causing test failures. It needs to be reverted then.
```suggestion
  if (use_remote_ui) {
```